ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -1076,6 +1192,11 @@ func (r *RayClusterReconciler) buildWorkerPod(instance rayv1.RayCluster, worker
 		serveLabel = true
 	}
 	pod := common.BuildPod(podTemplateSpec, rayv1.WorkerNode, worker.RayStartParams, headPort, autoscalingEnabled, creatorName, fqdnRayIP, serveLabel)
+
+	// Set multihost pod labels
+	podTemplateSpec.Labels[utils.RayNodeMultihostGroupKey] = strconv.FormatUint(uint64(multihostGroup), 10)
```

Comment:
Should we call this something like MultihostReplicaKey (and at other places) since it identifies one unique replica? "Group" might get confused with worker group.

Reply:
That sounds good to me, I changed instances of `RayNodeMultihostGroupKey` and `groupKey` to Replica.